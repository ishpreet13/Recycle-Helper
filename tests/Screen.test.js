import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import Screen from "../app/components/Screen";

describe("Testing Screen Component", () => {
  const wrapper = renderer.create(<Screen />);
  it("should render", () => {
    expect(wrapper.toJSON()).toBeTruthy();
  });
});

describe("<Screen />", () => {
  let tree;
  it("should render correctly", async () => {
    tree = renderer.create(<Screen />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
