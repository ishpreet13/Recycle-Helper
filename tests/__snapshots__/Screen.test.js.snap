// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Screen /> should render correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "flex": 1,
        "paddingTop": 20,
      },
      undefined,
    ]
  }
/>
`;
